client/sync.js

```diff
@@ -96,6 +96,75 @@ const startSync = (requester) => {
     return jsRecords
   }
 
+  const getBaseBookmarksOrder = (deviceId, platform) => {
+    let platformPrefix = `1.`
+    if (platform === `ios` || platform === `android`) {
+      platformPrefix = `2.`
+    }
+    platformPrefix += deviceId + `.`
+    return platformPrefix
+  }
+
+  const getBookmarkOrder = (prevOrder, nextOrder) => {
+    let prevOrderSplit = prevOrder.split(`.`)
```

Comment:
you can replace ` with ' for strings in JS in most of this code since ` is only needed for template strings. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals